ansible/roles/apache/templates/etc/sudoers.d/apache_webadmins.j2

```diff
@@ -0,0 +1,28 @@
+{# Copyright (C) 2014-2017 Maciej Delmanowski <drybjed@drybjed.net>
+ # Copyright (C) 2015-2017 Robin Schneider <ypid@riseup.net>
+ # Copyright (C) 2014-2017 DebOps <https://debops.org/>
+ # Copyright (C) 2020-2020 Sebastian Vaisov <sebastian@vaisov.lt>
+ # SPDX-License-Identifier: GPL-3.0-only
+ #}
+# {{ ansible_managed }}
+
+Cmnd_Alias APACHE_RELOAD =  /etc/init.d/apache2 reload, \
+                           /bin/systemctl reload apache2.service
+
+Cmnd_Alias APACHE_STATUS =  /etc/init.d/apache2 status, \
+                           /bin/systemctl status apache2.service
+
+Cmnd_Alias APACHE_STOP =    /etc/init.d/apache2 stop, \
+                           /bin/systemctl stop apache2.service
+
+Cmnd_Alias APACHE_START =   /etc/init.d/apache2 start, \
+                           /bin/systemctl start apache2.service
+
+Cmnd_Alias APACHE_RESTART = /etc/init.d/apache2 restart, \
```

Comment:
Not sure if we need legacy support like `/etc/init.d/apache2`?

Reply:
Remove